app/src/main/java/org/dhis2/data/dhislogic/DhisMapUtils.kt

```diff
@@ -0,0 +1,53 @@
+package org.dhis2.data.dhislogic
+
+import javax.inject.Inject
+import org.hisp.dhis.android.core.D2
+import org.hisp.dhis.android.core.common.FeatureType
+import org.hisp.dhis.android.core.common.Geometry
+import org.hisp.dhis.android.core.common.ValueType
+import org.hisp.dhis.android.core.dataelement.DataElement
+import org.hisp.dhis.android.core.enrollment.Enrollment
+import org.hisp.dhis.android.core.event.Event
+import org.hisp.dhis.android.core.program.ProgramStage
+
+class DhisMapUtils @Inject constructor(val d2: D2) {
```

Comment:
The idea is to move it to SDK,right?

Reply:
Yes, all classes in the dhislogic folder is for the sdk :)